crates/weaver_forge/codegen_examples/templates/registry/rust/weaver.yaml

```diff
@@ -0,0 +1,62 @@
+type_mapping:
+  int: i64
+  double: f64
+  boolean: bool
+  string: String
+  string[]: Vec<String>
+  template[string]: String          # Not yet properly handled in codegen
+  template[string[]]: Vec<String>   # Not yet properly handled in codegen
+
+templates:
+  - pattern: lib.rs.j2
+    # The following JQ filter extracts the id, type, brief, and prefix of groups matching the following criteria:
+    # - groups with an id starting with the prefix `registry.`
+    # - groups of the type `attribute_group`.
+    # - groups with a well-defined prefix.
+    # - groups with a non-empty list of attributes that are neither deprecated nor experimental.
+    # - groups are deduplicated by prefix.
+    # - groups are sorted by prefix.
+    filter: >
+      .groups
+      | map(select(.id | startswith("registry.")))
+      | map(select(.type == "attribute_group" and .prefix != null and .prefix != "")    
+        | { 
+          id, 
+          type, 
+          brief, 
+          prefix, 
+          attributes: (.attributes                                                      
```

Comment:
it depends on the SIG decision, but I'd generate experimental deprecated attributes

Reply:
Ok I will change the configuration, no change in the template is required.

Reply:
Changed the config.